java-utils/java-functions

```diff
@@ -120,94 +116,24 @@ _load_java_conf()
 }
 
 
-# Test default JRE/JVM roots if nothing is defined yet
-_set_java_home()
-{
-    local IFS=:
-    local jvmdir
-    local subdir
-    local subdirs
-
-    if [ -n "${JAVA_HOME}" ]; then
-	if [ -z "${JVM_ROOT}" ]; then
-	    JVM_ROOT=$(readlink -f "${JAVA_HOME}"/.. || :)
-	fi
-	return
-    fi
-
-    case "${_prefer_jre}" in
-	1|[Yy][Ee][Ss]|[Tt][Rr][Uu][Ee])
-	    subdirs=jre:java
-	    ;;
-	*)
-            subdirs=java:jre
-	    ;;
-    esac
-
-    for subdir in $subdirs; do
-	for jvmdir in $_jvmdirs; do
-	    JVM_ROOT="${jvmdir}"
-	    JAVA_HOME="${jvmdir}/${subdir}"
-	    if [ -d "${JVM_ROOT}" -a -d "${JAVA_HOME}" ]; then
-		_log "Using configured JVM_ROOT: ${JVM_ROOT}"
-		_log "Using configured JAVA_HOME: ${JAVA_HOME}"
-		return
-	    fi
-	done
-    done
-
-    unset JVM_ROOT
-    unset JAVA_HOME
-}
-
 # Set the java virtual machine
 set_jvm()
 {
     local IFS=:
     local cmd
     local cmds
 
-    _set_java_home
-
-    # use $JAVA_HOME if defined
-    if [ -n "${JAVA_HOME}" ]; then
-	return
+    if [ -z "${JAVA_HOME}" ]; then
+        _err "JAVA_HOME not configured or set"
```

Comment:
Users that have old java.conf would get this message. Would be nice to point them to some doc explaining how to fix this.